contracts/eosio.system/include/eosio.system/eosio.system.hpp

```diff
@@ -373,6 +378,19 @@ namespace eosiosystem {
 
    typedef eosio::multi_index< "finkeyidgen"_n, fin_key_id_generator_info >  fin_key_id_gen_table;
 
+   // A single entry storing a vector of names, each of which is a pattern that new account names
+   // are checked against (when the `newaccount` is called), in order to reject the creation
+   // of accounts whose name matches patterns in the blacklist.
+   struct [[eosio::table("accountnmbl"), eosio::contract("eosio.system")]] account_name_blacklist {
```

Comment:
Seems like `actblacklist` might be a better name.

Reply:
https://github.com/eosnetworkfoundation/eos-system-contracts/pull/189/commits/63dca594f5c9d377b97733515cf240ca76cc9b4f